src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cshtml

```diff
@@ -35,14 +35,19 @@
                                 <table class="govuk-table">
                                     <thead class="govuk-table__head">
                                         <tr class="govuk-table__row">
-                                            <th scope="col" class="govuk-table__header">Course name</th>
+                                            <th scope="col" class="govuk-table__header">Course</th>
                                             <th scope="col" class="govuk-table__header" style="width: 150px">Delivery type</th>
                                             <th scope="col" class="govuk-table__header" style="width: 130px">Start date</th>
                                         </tr>
                                     </thead>
                                     <tbody class="govuk-table__body">
                                         @foreach (var row in Model.RowGroups)
                                         {
+                                            <tr>
```

Comment:
This is different to the other 4 places where we've added it. Is that intention?

Reply:
example: https://github.com/SkillsFundingAgency/dfc-coursedirectory/pull/2154/files#diff-5e790bcfda982e2ded6fca2df647db03f838ff3f501f4135760b098d5817ccd9R29-R32

Reply:
Possibly need the right-aligned date too: https://github.com/SkillsFundingAgency/dfc-coursedirectory/pull/2154/files#diff-5e790bcfda982e2ded6fca2df647db03f838ff3f501f4135760b098d5817ccd9R65

Reply:
oversight on my behalf.